sherpa/astro/ui/utils.py

```diff
@@ -62,7 +59,7 @@ class Session(sherpa.ui.utils.Session):
     def __init__(self):
 
         self.clean()
-        sherpa.ui.utils.Session.__init__(self)
+        super().__init__()
```

Comment:
:+1:

Reply:
I have read conflicting information on "super considered harmful", but for most of our cases it isn't an issue (although the model class, and some of the plotting classes, do have multiple inheritance to worry about).